_posts/2022-05-31-Alarmeboard_mit_Jira_und_Cloudwatch.md

```diff
@@ -0,0 +1,369 @@
+
+### Kosten
+Mit diesem Gesamtbild vor Augen kommen wir schnell auf die Frage, ob das nicht ein bisschen teuer werden könnte.
+Denn am Ende soll sich die Investition ja auch wieder lohnen. 
+Dabei spielen zwei Faktoren eine wesentliche Rolle.
+Zum einen müssen wir betrachten wie viele Alarme bearbeitet werden müssen und wie stark das Team dadurch gebunden wird.
+Damit ist hier ein Faktor gegeben, den jedes Projekt für sich selbst beantworten muss.
+Aber die laufenden Kosten können wir hier zumindest grob überschlagen:
+* Lambdas verursachen nur zur Laufzeit kosten.
+  Selbst wenn wir hier deutlich überzogene 1.000 Aufrufe mit 15.000 ms Laufzeit und 512MB Speicher veranschlagen, kommen wir ohne Berücksichtigung des Freetiers auf gerade einem 0,12 USD Kosten pro Monat.
+* Für SNS und SQS gilt ähnliches.
+  Wenn man hier den Freetier von 1.000.000 Notifications pro Monat bei SNS und SQS verlässt, entstehen pro Request nur 0.0000005 USD an Kosten.
+  Das sollte bei der Summe der erwarteten Alarme nie einen nennenswerten Betrag ergeben.
+* Alarm-Metriken verursachen Kosten. 
+  Diese sollten aber nicht dieser Lösung angerechnet werde, da sie in den zu überwachenden Projekten sowieso bereits benötigt werden.
+
+Am Ende können wir sagen, dass die laufenden Kosten für die hier verwendeten Komponenten selbst jenseits des Freetiers zu vernachlässigen sind.
+
+# Automatisierung in Jira nutzen
+Für die Bearbeitung von Alarmen in Jira bietet sich ein Kanban Board spontan an.
+Für das Board lassen sich Automatisierungen einrichten, die die Arbeit mit den Alarmtickets weiter erleichtern.
+Folgende Ideen lassen sich mit wenigen Klicks umsetzen:
+
+**Erinnerung an Tickets, die nach 24 Stunden noch nicht bearbeitet wurden**
+
+Hierfür lassen wir eine Automatisierung geplant jede Stunde eine Suche ausführen, die nach allen Tickets sucht, die in unserem Alarme-Board länger als 24 Stunden im Status "New" geblieben sind.
+Das kann als JQL folgendermaßen aussehen:
+```jql
+createdDate <= -1d  AND statuscategory = "New" AND project = "Alarme"
+```
+Dabei ist es möglich die Bedingung der Regel auf Tickets einzuschränken, die seit dem letzten Lauf dazugekommen sind.
+
+Über den Then-Part der Regel lässt sich dann eine E-Mail auslösen, die an einen Verteiler gerichtet werden kann und direkt auf die noch zu bearbeitenden Tickets referenziert:
+```html
+Bitte den Alarm bearbeiten: <a href="https://karstadt.atlassian.net/browse/{{issue.key}}">{{issue.key}}</a>
+```
+
+**Projektbezogene Zuweisung von Tickets**
+
+In einigen Fällen mag es sinnvoll sein, dass bestimmte Tickets immer bei einem bestimmten Entwickler landen sollen.
+Über den Trigger "Vorgang erstellt", bekommt Jira mit, wenn ein neues Ticket angelegt wird.
+Damit lassen sich dann wieder Bedingungen verknüpfen.
+Zum Beispiel kann das Feld "Stichwort", in welches wir in der Lambda den Projektnamen geschrieben haben, nach einem bestimmten Wert durchsucht werden.
```

Comment:
Verstehe ich nicht ganz. Wie verknüpft sich das mit der Lambda? Ist das das Feld "Subject"?

Reply:
ja, ist das gleiche, je nachdem welche Sprache man in Jira einstellt. Ich habe das mal versucht klarer zu machen